templates/tonic-pyml-deployment.yaml

```diff
@@ -34,6 +34,8 @@ spec:
         resources:
           limits:
             memory: "1Gi"

```

Comment:
Ander once warned me that up to 2gb could be used on abnormally large models. @gandersteele does any of this change with the addition of the event synthesis feature